### ISSUE TITLE
Support recursive directories

### ISSUE THREAD

**Issue by SamJongenelen** [Augurk/Augurk.CommandLine#15]
Currently there is support for wildcards, but this does not seem to support recursion 
(https://github.com/Augurk/Augurk.CommandLine/blob/master/src/Augurk.CommandLine/Commands/PublishCommand.cs#L220)

**suggestion**:  use the minimatch pattern to support recursion? 

**workaround**:  wrap the Augurk.exe command in some powershell and implement in VSTS task: 

```Powershell
$children = Get-ChildItem "$(featureFilesDirectory)" -File -Filter "*.feature" -Recurse | select -ExpandProperty FullName 
$csFeatureFiles = $children -join ","

&"$(augurkLocation)"  publish --url "$(url)" --productName "$(productName)" --productDesc "$(productDesc)" --groupName "$(groupName)" --language "$(language)" --featureFiles $csFeatureFiles --cleargroup --branchName "$(branchName)" --embed
```

**Comment by jmezach** [comment]
@SamJongenelen You're right, we do not currently support this natively inside the command line. We do however support this in our Azure DevOps (VSTS) extension.

That being said I can imagine scenario's where it can be useful to have this inside the commandline itself. We also have an open issue regarding relative paths to the folder containing the feature files (#10), so we might consider combining these two issues

**Comment by SamJongenelen** [comment]
Yes. Currently I am unable to consume the VSTS extension, as we are using an on premise TFS :)

**Comment by jmezach** [comment]
@samjongenelen The extension should work with on-premises TFS as well. You should be able to install it through the Marketplace. Please let us know if you have any issues getting it installed.

**Comment by SamJongenelen** [comment]
Yes, however i don't have those credentials in the workplace

**Comment by jmezach** [comment]
I think you should be able to request installation so that someone can review your request and approve/disapprove it.
